deployments/Dockerfile

```diff
@@ -22,6 +22,7 @@ WORKDIR /opt/diwise
 COPY --from=builder --chown=1001 /app/cmd/iot-device-mgmt/iot-device-mgmt /opt/diwise
 COPY --chown=1001 assets/config/authz.rego /opt/diwise/config/authz.rego
 COPY --chown=1001 assets/config/devices.csv /opt/diwise/config/devices.csv
+COPY --chown=1001 assets/config/notifications.yaml /opt/diwise/config/notifications.yaml
```

Comment:
Måste de finnas default i dockerfilen? Räcker det inte att vi monterar in filen om vi behöver den?